src/Anemone/Foreign/Mempool.hs

```diff
@@ -47,6 +48,9 @@ foreign import ccall unsafe "anemone_mempool_free"
 foreign import ccall unsafe "anemone_mempool_total_alloc_size"
   totalAllocSize :: Mempool -> IO Int64
 
+foreign import ccall unsafe "anemone_is_pointer_aligned"
+  isPointerAligned :: Ptr Void -> IO Bool
```

Comment:
There isn't, but would be happy for you to add a function `Anemone.Foreign.Data.fromCBool :: CBool -> Bool`

You can compare against `0` with `CBool` too, it supports `Integral`.